CSS/style.css

```diff
@@ -0,0 +1,120 @@
+html, body {
```

Comment:
This looks really good! I would suggest being a bit more careful with your choice of selectors, as there are a lot of IDs used for styling here which are of a higher priority than classes so can make things difficult to change later on if the project gets more complicated. I would recommend adding classes to the elements youre targeting and using them for your css definitions instead.
In the case of any elements javascript is not targeting to perform logic, the IDs could actually be removed